pkg/api/v20230701preview/openshiftcluster.go

```diff
@@ -135,7 +138,7 @@ const (
 
 // ResourceReference represents a reference to an Azure resource.
 type ResourceReference struct {
-	// The fully qualified Azure resource id.
+	// The fully qualified Azure of an IP address resource.
```

Comment:
Seems like a small typo compared to what we upstreamed: https://github.com/Azure/azure-rest-api-specs/pull/25182/files#diff-9e37deed88559aa01045e43a7eb621ea865edb3a79cd0e90b8e947ba2879410cR1854